src/Exception/NotFoundException.php

```diff
@@ -8,6 +8,6 @@
 /**
  * No entry was found in the container.
  */
-interface NotFoundException extends ContainerException
+interface NotFoundException
```

Comment:
See https://github.com/container-interop/container-interop/issues/2 Indeed it could be included in the meta document.